### PR TITLE
Add Mozilla Blog Post

### PR DESCRIPTION
**Pull request by skade**

### CODE REVIEW

File: _posts/2017-09-30-mozilla.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "Firefox Quantum - using Rust for parallelized layouting"
+authors:
+  - florian
+---
+
+<center><img style="max-width: 100%" src="https://blog.mozilla.org/wp-content/uploads/2017/09/2017.25.09.FFQuantumBeta_Quantum-1400x770.png" /></center>
```
Comment (dns2utf8) [inline _posts/2017-09-30-mozilla.md]: I would prefer this to be hosted on our site

File: _posts/2017-09-30-mozilla.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+With the release of Firefox Quantum we are not done yet. WebRender is a project that is [currently being worked on](https://mozillagfx.wordpress.com/category/wr-newsletter/). WebRender is written in Rust and uses parallelism to render websites.
+
+## Get involved and test Mozilla Quantum!
+
+The Mozilla community is organizing testing events for Firefox Quantum.
+
+We invite you to help us test websites from your area, head over to the
+[Firefox Quantum Sprint
+website](https://firefoxsprint.mozilla.community/) to find an event.
```
Comment (dns2utf8) [inline _posts/2017-09-30-mozilla.md]: strange line break

File: _posts/2017-09-30-mozilla.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+authors:
+  - florian
+---
+
+<center><img style="max-width: 100%" src="https://blog.mozilla.org/wp-content/uploads/2017/09/2017.25.09.FFQuantumBeta_Quantum-1400x770.png" /></center>
+
+
+A year ago Mozilla announced [Project Quantum](https://medium.com/mozilla-tech/a-quantum-leap-for-the-web-a3b7174b3c12), an effort to create a next generation browser engine, leveraging new technologies from the [Servo project](https://servo.org/). With the planned release of Firefox Quantum on November 14th, several Rust components will be running within Firefox. Quantum CSS (aka "Stylo") uses parallelism to lay out pages, making Firefox the first browser to use multiple cores for rendering. These changes have landed in Firefox Beta as well as Firefox Developer Edition and can be tested right now! [See this blog post for details.](https://blog.mozilla.org/blog/2017/09/26/firefox-quantum-beta-developer-edition/) With Firefox Quantum you will also get a refreshed, modern, high performance user interface.
+
+Stay up to date and get notified when Firefox Quantum launches: https://www.mozilla.org/en-US/firefox/quantum/
```
Comment (dns2utf8) [inline _posts/2017-09-30-mozilla.md]: Link is not klickable

File: _posts/2017-09-30-mozilla.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+We invite you to help us test websites from your area, head over to the
+[Firefox Quantum Sprint
+website](https://firefoxsprint.mozilla.community/) to find an event.
+
+## Open development
+
+The Mozilla teams bringing Rust to Firefox have been busy writing about what they bring to Firefox. We assembled a collection of the most interesting blog posts for you!
+
+## Browser internals, mostly Rust
```
Comment (dns2utf8) [inline _posts/2017-09-30-mozilla.md]: h3? since it is a group inside the collection

File: _posts/2017-09-30-mozilla.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+*  [https://hacks.mozilla.org/2017/05/quantum-up-close-what-is-a-browser-engine/](Quantum Up Close: What is a browser engine?)
+
+* [https://hacks.mozilla.org/2017/08/inside-a-super-fast-css-engine-quantum-css-aka-stylo/](Inside a super fast CSS engine: Quantum CSS (aka Stylo))
+
+* [https://hacks.mozilla.org/2017/06/an-inside-look-at-quantum-dom-scheduling/](An inside look at Quantum DOM Scheduling)
+
+## General Engineering blog posts
```
Comment (dns2utf8) [inline _posts/2017-09-30-mozilla.md]: h3? same

File: _posts/2017-09-30-mozilla.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+* [https://blog.mozilla.org/firefox/fast-responsive-reliable-browser-multi-tab-age/](Fast, Responsive, Reliable: A Browser For the Multi-Tab Age)
+
+* [https://hacks.mozilla.org/2017/09/firefox-quantum-developer-edition-fastest-firefox-ever/](Firefox Quantum Developer Edition: the fastest Firefox ever with Photon UI and better tooling)
+
+* [https://hacks.mozilla.org/2017/06/designing-for-performance-a-data-informed-approach-for-quantum-development/](Designing for performance: A data-informed approach for Quantum development)
+
+* [https://hacks.mozilla.org/2017/09/developer-edition-devtools-update-now-with-photon-ui/](Developer Edition Devtools Update: Now with Photon UI)
+
+## Interesting Newsletters to read
```
Comment (dns2utf8) [inline _posts/2017-09-30-mozilla.md]: newsletter, lowercase and h3?

File: _posts/2017-09-30-mozilla.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+* [https://ehsanakhgari.org/tag/quantumflow](Quantum Flow project newsletter)
```
Comment (dns2utf8) [inline _posts/2017-09-30-mozilla.md]: This list is not seperated by empty lines like the others. I would make the others like this.
However the style sould be consistent

**Review by dns2utf8 — CHANGES_REQUESTED** [review]
made my remarks

**Comment by skade** [comment]
pushed the changes, thanks :)

**Comment by dns2utf8** [comment]
The push did not make it, can you send it again?

**Comment by dns2utf8** [comment]
PR is missing the asset `/assets/posts/2017-09-30-mozilla.png`

**Pull request merged**